custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -63,11 +63,13 @@ def person(self):
                 # 'owner_id': self._location.location_id,
                 'update': {
                     'aadhaar_number': self.patient_detail.paadharno,
+                    'age': self.patient_detail.page,
                     'age_entered': self.patient_detail.page,
                     'contact_phone_number': validate_number(self.patient_detail.pmob),
                     'current_address': self.patient_detail.paddress,
                     'current_address_district_choice': self.patient_detail.Dtocode,
                     'current_address_state_choice': self.patient_detail.scode,
+                    'dob_known': False,
```

Comment:
> I see. I think it would still be good to map True and False to ('true', 'false') (or ('yes', 'no')) because those are the conventions we've been using in the app.

https://github.com/dimagi/commcare-hq/pull/14296